roles/api-web/templates/start-container.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# usage: start-container.sh WEBAPP-NAME
+
+set -eu
+
+webapp=${1:-}
+if [ -z "$webapp" ]; then
+    echo 2>&1 "usage: start-container.sh WEBAPP-NAME"
+    exit 1
+fi
+if [ "$(echo $webapp | tr -d "A-Za-z0-9-_")" ]; then
+    echo 2>&1 "invalid webapp name: $webapp"
+    exit 1
+fi
+if [ ! -d /opt/docker/workspace/"$webapp" ]; then
+    echo 2>&1 "webapp does not exist: $webapp"
+    exit 1
+fi
+
+webapp_path=/opt/docker/workspace/"$webapp"
+cd $webapp_path &&
```

Comment:
what if $webapp contains a space in the directory? Will this affect the cd command?